src/lib/manifestationFactoryUtils.js

```diff
@@ -76,6 +76,7 @@ export function formatMaterialTypesToPresentation(materialTypeArray) {
  * @param manifestation
  * @returns {*|*[]}
  */
+//THIS ONE gets materialtype array
```

Comment:
Hvis du sletter den her, så slipper jeg for en merge conflict på manifestationFactoryUtils